<launch>
	<arg name="marker_size" 		default="8.8" />
	<arg name="max_new_marker_error" 	default="0.08" />
	<arg name="max_track_error" 		default="0.2" />

	<arg name="cam_image_topic" 		default="/camera/rgb/image_raw" />
	<arg name="cam_info_topic" 		default="/camera/rgb/camera_info" />
	<arg name="output_frame" 		default="/neck_center_body_yaw_frame" />

	<arg name="user_data_2_map" 		default="true" />
	<arg name="marker_topic" 		default="/ar_pose_marker" />
	<arg name="output_marker_topic" 	default="/marker_signal" />


    <node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false">
    	<param name="marker_size"           type="double" value="$(arg marker_size)" />
    	<param name="max_new_marker_error"  type="double" value="$(arg max_new_marker_error)" />
    	<param name="max_track_error"       type="double" value="$(arg max_track_error)" />
    	<param name="output_frame"          type="string" value="$(arg output_frame)" />

    	<remap from="camera_image"  		to="$(arg cam_image_topic)" />
    	<remap from="camera_info"   		to="$(arg cam_info_topic)" />
    </node>

	<node pkg="automap" name="marker2Map" type="marker2Map" output="screen"/>

</launch>
